chef/cookbooks/cinder/recipes/volume.rb

```diff
@@ -88,8 +88,16 @@ def make_volume(node, backend_id, volume)
 
   return if volume_exists(volname)
 
-  unclaimed_disks = BarclampLibrary::Barclamp::Inventory::Disk.unclaimed(node)
-  claimed_disks = BarclampLibrary::Barclamp::Inventory::Disk.claimed(node, "Cinder")
+  claim_string = "Cinder"
```

Comment:
I would prefer snakecase claim strings as we anyway display some I18n key within crowbar.